.travis.yml

```
@@ -1,5 +1,5 @@
 language: node_js
-node_js: 'lts/dubnium'
+node_js: stable
```

We may as well set this explicitly as well

```suggestion
node_js: 'lts/carbon'
```